docs/decisions/ADR-007-Edge-Authentication.md

```
@@ -46,149 +38,87 @@ JWT 검증 = CPU 연산 (로컬)
-│  │(UI Aggreg.)│    │   (Saga)    │    │(Token Iss.)│            │
-│  └────────────┘    └─────────────┘    └────────────┘            │
-│      │                  │                                       │
-│      ▼                  ▼                                       │
-│  ┌─────────────────────────────────────────────────────────┐    │
-│  │  User / Product / Order / Payment Services              │    │
-│  │  • Consumes Internal Token only                         │    │
-│  │  • Focus on business logic                              │    │
-│  └─────────────────────────────────────────────────────────┘    │
-└─────────────────────────────────────────────────────────────────┘
-```
+**구성 요소**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)